src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -7,14 +7,1284 @@
+     */
+    RELAY_OFF_V_BAD: boolean;
+
+    /**
+     * Turn off on exceeding the differential current limit.
+     */
+    RELAY_OFF_DIFF_BAD: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T1`.
+     */
+    RELAY_OFF_LIM_TARIFF_0: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T2`.
+     */
+    RELAY_OFF_LIM_TARIFF_1: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T3`.
+     */
+    RELAY_OFF_LIM_TARIFF_2: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T4`.
+     */
+    RELAY_OFF_LIM_TARIFF_3: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T1`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_0: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T2`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_1: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T3`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_2: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T4`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_3: boolean;
+
+    /**
+     * Turn on when `cos φ` is normalized.
+     */
+    RELAY_ON_PF_MIN: boolean;
+
+    /**
+     * Turn off on `cos φ`.
+     */
+    RELAY_OFF_PF_MIN: boolean;
+
+    /**
+     * Turn on after timeout.
+     */
+    RELAY_ON_TIMEOUT: boolean;
+
+    /**
+     * Turn on based on saldo.
+     */
+    RELAY_ON_SALDO: boolean;
+
+    /**
+     * Turn off based on saldo.
+     */
+    RELAY_OFF_SALDO: boolean;
+
+    /**
+     * Turn off based on saldo with condition.
+     */
+    RELAY_OFF_SALDO_SOFT: boolean;
+}
+
+export interface ISpeedOptoPortOperatorParameter {
+    /**
+     * Baud rate of the PLC (UART_0): `2400` or `9600`.
+     *
+     * | Value | Baud Rate |
+     * |-------|-----------|
+     * | 0     | 9600      |
+     * | 2     | 2400      |
+     * | 4     | 9600      |
+     */
+    plc: typeof baudRates.RATE_2400 | typeof baudRates.RATE_9600,
+
+    /**
+     * Baud rate of the optoport (UART_1): `2400` or `9600`.
+     *
+     * | Value | Baud Rate |
+     * |-------|-----------|
+     * | 0     | 2400      |
+     * | 2     | 2400      |
+     * | 4     | 9600      |
+     */
+    optoport: typeof baudRates.RATE_2400 | typeof baudRates.RATE_9600
+}
+
+export interface ITypeMeterOperatorParameter {
+    /**
+     * Consideration of transformation ratio.
+     * `0` - without transformation ratio, `1` - with transformation ratio.
+     */
+    TRANSFORMATION_RATIO: boolean;
+
+    /**
+     * Meter type.
+     * `1` - R-type meter.
+     */
+    METER_TYPE_R: boolean;
+
+    /**
+     * Reactive energy accumulation type.
+     * `0` - reactive energy accumulation by quadrants `Q1, Q2, Q3, Q4`;
+     * `1` - reactive energy accumulation by `R+, R-`.
```

Comment:
I would prefer to indicate separeate values instead

```suggestion
     * `1` - reactive energy accumulation by `R+`, `R-`.
```

and in other places

Reply:
applied